helmfile.d/snippets/derived.gotmpl

```diff
@@ -105,8 +105,6 @@ environments:
             enabled: true
             address: {{ $keycloakBaseUrl }}
             adminPassword: {{ $a | get "keycloak.adminPassword" $v.otomi.adminPassword }}
-          kured:
```

Comment:
Perhaps we should define a migration that sets enabled flag if an upgrade happens. In this way we ensure that there is no impact to the current Otomi instances. What do you think ?